### PR TITLE
adding additional linux ubuntu install instructions

### PR DESCRIPTION
**Pull request by adrian-plata**
Live: https://docs.docker.com/install/linux/docker-ee/ubuntu/#install-from-a-package
Preview: https://deploy-preview-9934--docsdocker.netlify.com/install/linux/docker-ee/ubuntu/#install-from-a-package

Signed-off-by: Adrian Plata <adrian.plata@docker.com>

### CODE REVIEW

File: install/linux/docker-ee/ubuntu.md [hunk 1]
```diff
@@ -247,13 +247,20 @@ To upgrade Docker Engine - Enterprise, follow the steps below:
...
-    `ubuntu/x86_64/stable-<VERSION>` and download the `.deb` file for the
-    Docker Engine - Enterprise version and architecture you want to install.
+    `ubuntu/dists/<distribution>/pool/stable-<VERSION>` and download the `.deb`
+    file(s) for the Docker Engine - Enterprise version and architecture you want
+    to install.
+
+    > **Note:**
+    > For newer versions of Docker, you will download three files. They
+    > are `docker-ee-cli_<version>.deb`, `containerd.io_<version>.deb`, and
+    > `docker-ee_<version>.deb`.
```
Comment (lena-larionova) [inline install/linux/docker-ee/ubuntu.md]: Can we specify which version onward?

Comment (c4carlton) [reply install/linux/docker-ee/ubuntu.md]: These are the latest for the respective component.

2019-10-18T02:59:30Z | 20027634 | containerd.io_1.2.10-3_amd64.deb

2019-11-14T19:07:30Z | 101666388 | docker-ee-cli_19.03.5~3-0~ubuntu-bionic_amd64.deb

2019-11-14T19:07:30Z | 24839502 | docker-ee_19.03.5~3-0~ubuntu-bionic_amd64.deb

File: install/linux/docker-ee/ubuntu.md [hunk 2]
```diff
@@ -247,13 +247,20 @@ To upgrade Docker Engine - Enterprise, follow the steps below:
...
 1.  Go to the Docker Engine - Enterprise repository URL associated with your
     trial or subscription in your browser. Go to
-    `ubuntu/x86_64/stable-<VERSION>` and download the `.deb` file for the
-    Docker Engine - Enterprise version and architecture you want to install.
+    `ubuntu/dists/<distribution>/pool/stable-<VERSION>` and download the `.deb`
+    file(s) for the Docker Engine - Enterprise version and architecture you want
+    to install.
+
+    > **Note:**
+    > Starting with 19.03, you will download three `.deb` files. They
```
Comment (lena-larionova) [inline install/linux/docker-ee/ubuntu.md]: "Will download" makes it sound like 19.03 is a future release. I'd rephrase to something like "you need to" or "you have to".
